docs/content/en/main/concepts/syncer.md

```diff
@@ -164,36 +195,41 @@ You can run the syncer in a kind cluster for development.
    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/kubernetes-sigs/kind/main/site/static/examples/kind-with-registry.sh)"
    ```
 
-1. Install `ko`:
-
-    ```sh
-    go install github.com/google/ko@latest
+1. From the kcp root directory:
+    ```bash
+    export KUBECONFIG=.kcp/admin.kubeconfig
     ```
 
-1. Build image and push to the local registry integrated with `kind`:
+1. Create a location workspace and immediately enter it:
 
     ```sh
-    KO_DOCKER_REPO=localhost:5001 ko publish ./cmd/syncer -t <image tag>
+    $ kubectl kcp workspace create location --enter
+    Workspace "location" (type root:organization) created. Waiting for it to be ready...
+    Workspace "location" (type root:organization) is ready to use.
+    Current workspace is "root:location" (type "root:organization").
     ```
 
-    By default `ko` will build for `amd64`. To build for `arm64`
-    (e.g. apple silicon), specify `--platform=linux/arm64`.
+1. To create the synctarget and use the image pushed to the local registry, supply `<image name>` to the
+   `kcp workload sync` plugin command, where `<image name>` was captured in the build steps, above.
 
+    ```sh
+    kubectl kcp workload sync kind --syncer-image <image name> -o syncer.yaml
+    ```
+    
 1. Create an organisation and immediately enter it:
```

Comment:
```suggestion
1. Create a second workspace for your workloads and immediately enter it:
```

Reply:
In fact the workspaces we create here are `Organization` workpaces (and will be until the workspace refactoring ends up), as shown in the `create` command output.
But here we just use them as normal workspaces.
So maybe it's better to not mention organizations at all ?